cherab/openadas/parse/adf15.py

```diff
@@ -67,11 +67,18 @@ def parse_adf15(element, charge, adf_file_path, header_format=None):
         # use simple electron configuration structure for hydrogen-like ions
         if header_format == 'hydrogen' or element == hydrogen:
             config = _scrape_metadata_hydrogen(file, element, charge)
-        elif header_format == 'hydrogen-like' or element.atomic_number - charge == 1:
+        elif header_format == 'hydrogen-like':
             config = _scrape_metadata_hydrogen_like(file, element, charge)
+        elif element.atomic_number - charge == 1:
+            config = _scrape_metadata_hydrogen_like(file, element, charge)
+            if not config:  # try hydrogen header (works for 'bnd' files)
+                config = _scrape_metadata_hydrogen(file, element, charge)
```

Comment:
I have not found any files in ADAS other than those with the "bnd" suffix affected by this issue. Now I did an explicit check on the file name, but since ADAS may update the "bnd" files in the future, the parser tries to read the metadata in "hydrogen-like" format first, and if that fails, then for "bnd" files, it reads the metadata in "hydrogen" format.